apps/src/applab/debugArea.js

```diff
@@ -0,0 +1,236 @@
+/* jshint
+ funcscope: true,
+ newcap: true,
+ nonew: true,
+ shadow: false,
+ unused: true,
+
+ maxlen: 90,
+ maxparams: 3,
+ maxstatements: 200
+ */
+/* global $ */
+
+var applabMsg = require('./locale');
+var dom = require('../dom');
+var utils = require('../utils');
+
+var debugArea = module.exports;
+
+/** @type {jQuery} */
+var rootDiv_ = null;
```

Comment:
did you consider making debugArea an instantiable class, so that these could be members instead of globals? I was very happy to see that pattern in netsim, but I get that that pattern is not being used a lot in applab.